components/hr-solution/SeamlessIntegration.vue

```diff
@@ -0,0 +1,194 @@
+        class="btn button section__button mt-4 mb-4 mb-lg-0 mx-auto"
+        >Create Form</a
+      >
+    </div>
+    <!-- Icons For Large Devices -->
+    <div class="row sub-container d-none d-lg-flex justify-content-between">
+      <div
+        class="space-container d-flex justify-content-between"
+        v-for="(icon, index) in iconsForLargeScreen"
+        :key="index"
+      >
+        <div
+          class="d-flex justify-content-center align-items-start"
+          :class="{ 'align-items-end': isOdd(index) }"
+        >
+          <div class="icon-container">
+            <nuxt-img :src="icon.src" :alt="icon.alt" class="icon" />
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- Icons For Small Devices -->
+    <div class="row sub-container d-flex d-lg-none justify-content-between">
+      <div
+        class="space-container d-flex justify-content-between"
+        v-for="(icon, index) in iconsForSmallScreen"
+        :key="index"
+      >
+        <div
+          class="d-flex justify-content-center align-items-start"
+          :class="{ 'align-items-end': !isOdd(index) }"
+        >
+          <div class="icon-container">
+            <nuxt-img :src="icon.src" :alt="icon.alt" class="icon" />
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      iconsForLargeScreen: [
+        {
+          src: '/hr-solution/SeamlessIntegration/salesforce.png',
+          alt: 'salesforce',
+        },
+        {
+          src: '/hr-solution/SeamlessIntegration/slack.png',
+          alt: 'slack',
+        },
+        {
+          src: '/hr-solution/SeamlessIntegration/notion.png',
+          alt: 'slack',
+        },
+        {
+          src: '/hr-solution/SeamlessIntegration/hubspot.png',
+          alt: 'hubspot',
+        },
+        {
+          src: '/hr-solution/SeamlessIntegration/calendar.png',
+          alt: 'calendar',
+        },
+        {
+          src: '/hr-solution/SeamlessIntegration/g-sheet.png',
+          alt: 'g-sheet',
+        },
+        {
+          src: '/hr-solution/SeamlessIntegration/calendly.png',
+          alt: 'calendy',
+        },
+        {
+          src: '/hr-solution/SeamlessIntegration/g-drive.png',
+          alt: 'g-drive',
+        },
+        {
+          src: '/hr-solution/SeamlessIntegration/discord.png',
+          alt: 'discord',
+        },
+        // ...more icons
+      ],
+      iconsForSmallScreen: [
+        {
+          src: '/hr-solution/SeamlessIntegration/hubspot.png',
+          alt: 'hubspot',
+        },
+        {
+          src: '/hr-solution/SeamlessIntegration/calendar.png',
+          alt: 'calendar',
+        },
+        {
+          src: '/hr-solution/SeamlessIntegration/g-sheet.png',
+          alt: 'g-sheet',
+        },
+        {
+          src: '/hr-solution/SeamlessIntegration/calendly.png',
+          alt: 'calendy',
+        },
+        // ...more icons
```

Comment:
instead of using separate icons for both the screens will it be possible to only show fewer icons from the iconsforLargerscren itself?

Reply:
1. Yes we can do that if we want random four icons to show on small screen from data-iconsforLargerscren.
2. If we want specific icon to show there then we have to define it seperately (i.e. iconsForSmallScreen).